content/docs/start/data-and-model-versioning.md

```diff
@@ -165,7 +165,7 @@ If you've run `dvc push`, you can delete the cache (`.dvc/cache`) and
 
 ```dvc
 $ rm -rf .dvc/cache
-$ rm -f data/data.xml
+$ rm -Force data/data.xml
```

Comment:
This doesn't seem to be a Windows command though

![image](https://user-images.githubusercontent.com/1477535/152184392-8502d7bc-727e-4128-9529-f8842a2f6417.png)